src/components/tags/ConfirmableEditTagButton.js

```diff
@@ -0,0 +1,78 @@
+import React, { useContext, useRef, useState } from "react";
+import { Modal, Button, Row } from "react-bootstrap";
+import { BsGearFill } from "react-icons/bs";
+import "./ConfirmableEditTagButton.css";
+import { TagContext } from "./TagProvider";
+
+/**
+ * ConfirmableEditTagButton component - renders as an "Edit" button that, when clicked, renders a react-bootstrap Modal prompting the user to edit their tag and then confirm that they truly want to edit that tag.
+ *
+ * PROPS:
+ *  tag <Object> - the tag object, itself. Has a label and id property.
+ *  prompt (optional) <String> - The text to display as the confirmation text in the modal.
+ *    default value (if nothing passed in props) is "Are you sure you want to edit this item?"
+ */
+export const ConfirmableEditTagButton = (props) => {
+  const { updateTag } = useContext(TagContext);
+  const labelRef = useRef("");
+
+  const confirmEditPrompt = props.prompt || "Edit this Tag";
+
+  const [isEditing, setIsEditing] = useState(false);
+
+  const handleConfirmUpdate = () => {
+    updateTag({ id: props.tag.id, label: labelRef.current.value });
+    setIsEditing(false);
+  };
+
+  return (
+    <>
+      <div onClick={(e) => e.preventDefault()}>
+        <Button
+          className="border-0 bg-white text-dark"
+          onClick={() => setIsEditing(true)}
+        >
+          <BsGearFill style={{ fontSize: "36px" }} />
+        </Button>
+      </div>
+      {/* Need to change to className to confirmableEditModel */}
```

Comment:
I guess we can go ahead and change that className right? Since you did set that border none rule in the css file for this component